README.md

```diff
@@ -73,6 +73,41 @@ public function getFlexiblePreset(): ?string
 }
 ```
 
+### Custom fields per template
+
+You might want to provide your users with different fields based on the chosen template.  
+For instance, let's say you provide 3 templates: `default`, `homepage` and `news`.
```

Comment:
Ik vind het in documentatie altijd wel fijn als hier ook voorbeelden met een streepje of underscore tussen staan. Dan hoeft een dev er niet proefondervindelijk achter te komen wat de restricties van een template naam zijn.

Reply:
Dat is een heel goed punt!